tox.ini

```diff
@@ -13,3 +13,19 @@ deps =
   numpy
 commands =
   pytest {posargs}
+
+[testenv:cloud_tests]
+passenv =
+    MARQO_CLOUD_URL
+    MARQO_API_KEY
+    MARQO_URL
+    TEST_INDEX_SUFFIX
```

Comment:
make comments in this file explaining:
- why there is both `MARQO_URL` and `MARQO_CLOUD_URL`
- what `TEST_INDEX_SUFFIX` does and what its structure is 
-  which of these env vars are required and which are optional?